src/modules/CSADigitizer/README.md

```diff
@@ -19,6 +19,8 @@ Alternatively a custom impulse response function can be provided by using the `c
 Noise can be applied to the individual bins of the output pulse, drawn from a normal distribution.
 For the amplified pulse signal, alongside the Time-of-Arrival either the determined Time-over-Threshold, or the integral of the amplified pulse can be stored in the `PixelHit`.
 
+The values stored in `PixelHit` depend on the ToA and ToT settings. If a ToA clock is defined, then `local_time` will be stored in clock cycles, else in time units. If a ToT clock is defined, then `signal` will be the amount of ToT cycles the pulse is above the threshold, else it will be the integral of the amplified pulse. 
```

Comment:
I think line 20 can then be removed, as some information is repeated.

Reply:
Agreed, updated it.